code/modules/clothing/modular_armor/attachments/modules.dm

```diff
@@ -250,38 +250,12 @@
 	variants_by_parent_type = list(/obj/item/clothing/suit/modular/xenonauten = "mod_ff_xn")
 
 /obj/item/armor_module/module/chemsystem
-	name = "Vali chemical enhancement module"
-	desc = "Designed for mounting on modular armor. This experimental module runs on green blood taken from xenos with harvester class weapons; Green blood heals the user and boosts any chems in the suit injection system. \nUse the suit menu to connect harvester class weapons, control the injection system, find chem boost information, and more."
+	name = "Vali Health Scan System"
+	desc = "Designed for mounting on modular armor. This module gives an accurate and immediate readout of the wearer's vitals to them."
 	icon = 'icons/mob/modular/modular_armor_modules.dmi'
 	icon_state = "mod_chemsystem"
 	item_state = "mod_chemsystem_a"
 	slot = ATTACHMENT_SLOT_MODULE
-	///Lets us keep track of what icon state we're in
-	var/chemsystem_is_active = FALSE
-
-/obj/item/armor_module/module/chemsystem/on_attach(obj/item/attaching_to, mob/user)
-	. = ..()
-	var/datum/component/chem_booster/chemsystem = parent.AddComponent(/datum/component/chem_booster)
```

Comment:
preeeetty sure you just made the module do nothing.
You want to either plug in the chem booster component with no chems, or just make it give you the health scan action
/datum/action/suit_autodoc/scan

Reply:
ty, sorry, i reference valk for that but figured the healthscan was integrated into the splinting/or tricord/tram component on that and i missed it

Reply:
noted and to be fixed, i'll come back to this tomorrow though, as it is very late. appreciated, again.